File: Placements.DataAccess/placement/Models/Jobinterviewround.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Jobinterviewround
{
    public long Id { get; set; }

    public long? JobPostingId { get; set; }

    public string? Name { get; set; }

    public string? Description { get; set; }

    public int? Priority { get; set; }

    public virtual Jobposting? JobPosting { get; set; }

    public virtual ICollection<Jobinterviewpanel> Jobinterviewpanels { get; set; } = new List<Jobinterviewpanel>();

    public virtual ICollection<JobpostStudentround> JobpostStudentrounds { get; set; } = new List<JobpostStudentround>();
}

File: web-app/src/app/services/types/Jobinterviewround.ts
import { JobPostingList } from "src/app/features/company-menu/company-job-details/company-job-details-model";
import { Jobposting } from "./Jobposting";
import { JobpostStudentround } from "./JobpostStudentround";
import { Calendarevent } from "./Calendarevent";

export type Jobinterviewround = {
  Id: number;
  JobPostingId?: number;
  Name?: string;
  Description?: string;
  Priority?: number;
  EventId: number;
  JobPosting: Jobposting;
  JobpostStudentrounds: JobpostStudentround[];
  Event: Calendarevent;
  OrgId: number;
};

export type PatchJobinterviewround = Jobinterviewround | {};
